src/__tests__/route.test.js

```diff
@@ -5,4 +5,19 @@ describe('route', function() {
   it ('can handle empty routes', function() {
     route(API({ baseURL: '' }))
   })
+
+  it ('extends each route with configuration settings', function() {
+    let api = API({
+      description: 'My API',
+      baseURL: 'http://example.com'
+    })
+
+    route(api, { users: { read: { path: 'users' } } })
+
+    api.users.read.config.should.have.property('path', 'users')
+
+    for (var i in api.config) {
```

Comment:
do you not have foreach? You're using object.assign elsewhere, so I would assume...